packages/flutter/lib/src/widgets/app.dart

```diff
@@ -462,10 +484,19 @@ class _WidgetsAppState extends State<WidgetsApp> implements WidgetsBindingObserv
       child: new Localizations(
         locale: widget.locale ?? _locale,
         delegates: _localizationsDelegates.toList(),
-        child: new Title(
-          title: widget.title,
-          color: widget.color,
-          child: result,
+        child: new Builder(
+          builder: (BuildContext context) {
+            String title = widget.title;
```

Comment:
add a comment saying that we use a builder because we need a context that includes the Localizations widget